src/honey/sql.cljc

```diff
@@ -419,29 +425,30 @@
 (defn- sqlize-value [x] (p/sqlize x))
 
 (defn- param-value [k]
-  (if (contains? *params* k)
-    (get *params* k)
-    (throw (ex-info (str "missing parameter value for " k)
-                    {:params (keys *params*)}))))
+  (let [{:keys [params]} *options*]
+    (if (contains? params k)
+      (get params k)
+      (throw (ex-info (str "missing parameter value for " k)
+                      {:params (keys params)})))))
 
 (defn- ->param [k]
   (with-meta (constantly k)
     {::wrapper
      (fn [fk _] (param-value (fk)))}))
 
 (defn ->numbered [v]
-  (let [n (count (swap! *numbered* conj v))]
+  (let [{:keys [numbered]} *options*
+        n (count (swap! numbered conj v))]
     [(str "$" n) (with-meta (constantly (dec n))
                    {::wrapper
-                    (fn [fk _] (get @*numbered* (fk)))})]))
+                    (fn [fk _] (get @numbered (fk)))})]))
 
 (defn ->numbered-param [k]
-  (let [n (count (swap! *numbered* conj k))]
+  (let [{:keys [numbered]} *options*
+        n (count (swap! numbered conj k))]
     [(str "$" n) (with-meta (constantly (dec n))
                    {::wrapper
-                    (fn [fk _] (param-value (get @*numbered* (fk))))})]))
-
-(def ^:private ^:dynamic *formatted-column* (atom false))
```

Comment:
Interesting that this became unused at some point...

Reply:
I looked and couldn't find where the additional uses had disappeared -- and no tests failed so I suspect whatever this was introduced to solve got reimplemented a different way and this var was just never removed.